packages/utils/src/lib/is-date.ts

```
@@ -0,0 +1,9 @@
+/**
+ * Check if a value is a valid Date.
+ *
+ * @param value - The value to check.
+ * @returns {boolean} - True if the value is a valid Date, otherwise false.
+ */
+export function isDate(value: any): value is Date {
+	return value !== null && !isNaN(new Date(value).valueOf());
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)